cluster-manifests/kube-system/container-azm-ms-agentconfig.yaml

```diff
@@ -1,45 +1,164 @@
-apiVersion: v1
 kind: ConfigMap
-metadata:
-  name: container-azm-ms-agentconfig
-  namespace: kube-system
```

Comment:
Please keep the metadata at the top.  I know it's at the bottom in the other, but that's unstandard for a yaml manifest.

Reply:
done

Reply:
Thanks.